CMakeLists.txt

```
@@ -15,3 +25,4 @@ target_link_libraries(mcpelauncher-extract-lib ${LIBZIP_LIBRARIES})
 add_executable(mcpelauncher-extract src/main.cpp include/mcpelauncher/minecraft_extract_utils.h)
 target_link_libraries(mcpelauncher-extract mcpelauncher-extract-lib)
 
+install(TARGETS mcpelauncher-extract DESTINATION ${CMAKE_INSTALL_BINDIR})
```

For consistency with the other repositories of minecraft-linux I prefer to change this to
```suggestion
install(TARGETS mcpelauncher-extract DESTINATION bin)
```
and remove the rest of this change.

Otherwise needs to be changed in all submodules, that needs multiple PR's.